index.html

```diff
@@ -268,27 +268,27 @@ <h3>I'm always <br> interested in hearing <br> about new projects, <br> so if yo
             <ul class="list2">
                 <li>
                     <a class="social2item" href="https://twitter.com/Ouail_Laarabi" target=”_blank”>
-                        <img src="img/twitter.png">
+                        <img src="img/twitter.png" alt="twitter_icon">
                     </a>
                 </li>
                 <li>
                     <a class="social2item" href="https://www.linkedin.com/in/ouail-laarabi-53203b250/" target=”_blank”>
-                        <img src="img/Linkedin icon.png">
+                        <img src="img/Linkedin icon.png" alt="linkdin_icon">
                     </a>
                 </li>
                 <li>
                     <a class="social2item" href="#">
-                        <img src="img/medium.png">
+                        <img src="img/medium.png" alt="medium_icon">
                     </a>
                 </li>
                 <li>
                     <a class="social2item" href="https://github.com/Ouail-01" target=”_blank”>
-                        <img src="img/github.png">
+                        <img src="img/github.png" alt="github_icon">
                     </a>
                 </li>
                 <li>
                     <a class="social2item" href="#">
-                        <img src="img/angellist.png">
+                        <img src="img/angellist.png" alt="angellist_icon">
```

Comment:
- Good job making changes to improve the accessibility of your site 👏 👏. But according to the project requirements, ```Each point on the list needs to include the result of your evaluation:
If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
If you needed to change something, [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).
Add your report to the description of your pull request.```
- Please ensure that each of the points you have checked for has an evaluation result. 👍 👍